public/components/custom_panels/panel_modules/visualization_container/visualization_container.scss

```diff
@@ -34,6 +34,43 @@
   text-align: center;
 }
 
+.metricVis {
+  #metricsEditInline {
+    /* for ... reasons (?), margin here causes
+       other rules to activate... providing
+       a more-correct display.
+     */
+    margin: 0 0px;
```

Comment:
i didn't understand the comment, could you explain a bit?

Reply:
Without any other context, this CSS looks like a no-op.  Before/after this CSS the calculated styles margin is "already" 0 0.

I only discovered this CSS when hacking on the styles, and accidentally applied this CSS to the element.   Suddenly many of the other style issues I was encountering disappeared.